src/curated-corpus/pages/CustomSectionsPage/CustomSectionsPage.tsx

```diff
@@ -50,6 +39,7 @@ export const CustomSectionsPage: React.FC = (): JSX.Element => {
     notifyOnNetworkStatusChange: true,
     variables: {
       scheduledSurfaceGuid: currentScheduledSurfaceGuid,
+      createSource: 'MANUAL',
```

Comment:
lets leverage the generated type `ActivitySource` here, so `ActivitySource.MANUAL`